lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :resources do
+  desc "Update resources with nil quantity"
+  task update_resource_quantity: :environment do
+    Resource.all.each do |r|
```

Comment:
When I read `r`, I have to think about what it is referring to. Please use meaningful words, ie `resource`, that helps a lot to understand the code, when you read it later on.